content/collections/how-to/manage-custom-terms-type.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Manage a custom terms type
+weight: 6
+---
+
+# How to manage a custom terms type
+
+When tracking terms and conditions across different services, you might encounter situations where a service uses terms types that aren't yet supported by Open Terms Archive. 
+This guide will help you handle these custom terms types effectively, whether you're working on a new collection or expanding an existing one.
+
+## Understanding the validation
+
+Open Terms Archive validates terms types to ensure consistency and quality across collections. When you encounter a validation error like:
+
+```shell
+   1) Service declarations validation
+       <service_name>
+         valid declaration schema:
+      Error:
+
+ data/terms must be equal to one of the allowed values (in entire file)
+```
+
+This means the terms type you're trying to use isn't in the official list of supported types.
```

Comment:
Is there no way to improve the error message rather than explaining it in documentation?